README.md

```diff
@@ -27,19 +27,14 @@ producing complex, multi-container applications.
 2. Get a Vagrantfile for your box. Users of the
 [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) should download a [Vagrantfile from the repository](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components).
 
-3. Enable your desired service(s) in [Vagrantfile](Vagrantfile) as:
+3. Enable the plugin as:
 
-        config.servicemanager.services = 'openshift'
+        config.vm.provision = 'servicemanager'
 
-   *Note*
-
-   * `docker` is default service and does not require above configuration.
-   * Enable multiple services as comma separated list. Eg: 'docker, openshift'
-
-5. Start the ADB using `vagrant up`. Users of the ADB may wish to consult the
+4. Start the ADB using `vagrant up`. Users of the ADB may wish to consult the
 [Installation Documentation](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst).
 
-6. Run the plugin to get environment variables and certificates:
+5. Run the plugin to get environment variables and certificates:
```

Comment:
FYI, instead of manually numbering use all the time '1.'. Markdown will sort out the numbering. Explicitly numbering defeats the whole purpose, since one needs to manually adjust, especially if you insert an item.